docs/en-US/troubleshooting/version.md

```diff
@@ -0,0 +1,35 @@
+---
+category: 5. Troubleshooting
+order: 3
+title: Versioning system used by VVV
+description: How VVV project manage the various version and upgrades for the users
+permalink: /docs/en-US/troubleshooting/versioning-system/
```

Comment:
This should be under Introduction alongside the Governance page

Reply:
moved to keeping-up-date file